go/vt/worker/split_clone.go

```diff
@@ -1348,7 +1348,9 @@ func (scw *SplitCloneWorker) createKeyResolver(td *tabletmanagerdatapb.TableDefi
 	}
 
 	if *useV3ReshardingMode {
-		return newV3ResolverFromTableDefinition(scw.keyspaceSchema, td)
+		// @bramos: this will surely return an error at runtime which is what we want
+		// since we exclusively use LegacySplitClone
```

Comment:
Why not preemptively error out / panic here in that case?

Reply:
This was mostly because I was interested in updating the callsites for this function rather than preventing vtworker subcommands all together(note: we won't be using `split_clone` for merges). Come to think of it, it might be worth just panic-ing if we try to use one of these unsupported subcommands. Updates incoming.